src/main/resources/db/migration/dbchangelog.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
As we agreed during the last meeting, please group database changes in separate files